gulpfile.js

```diff
@@ -82,6 +83,58 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  var path = require('path');
+  var rootDir = 'dist';
+
+  var config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      rootDir + '/1.0/index.html',
```

Comment:
Yup, @arthurevans is right. Good catch! `dynamicUrlToDependencies` lets you keep track of which dependencies uniquely define a given URL, to ensure that if any of those underlying dependencies change, the cache entry for that URL will be updated.

For some setups, it could be easier to use globbing + a few constants to generate this mapping instead of trying to maintain it by hand. See, for example, https://github.com/GoogleChrome/ioweb2015/blob/master/gulp_scripts/service-worker.js#L24